src/main/java/taxcalculator/validation/ValidateInput.java

```diff
@@ -0,0 +1,70 @@
+package taxcalculator.validation;
+
+import static taxcalculator.constants.Commands.NAME_CMD;
+import static taxcalculator.constants.Regex.IS_NUMERIC_REGEX;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+import taxcalculator.enums.Type;
+
+public class ValidateInput {
+  public static void checkNumeric(String str) throws Exception {
+    if (!str.matches(IS_NUMERIC_REGEX)) {
+      throw new Exception("Enter integral value only");
+    }
+  }
+
+  public static boolean isEndInput(String line, Type type) {
+    return line.equals("") && Objects.nonNull(type);
```

Comment:
check java validation library.
validation and utils separate.
https://www.baeldung.com/javax-validation-method-constraints